src/plugins/partner_sku_reviews.coffee

```diff
@@ -125,19 +125,25 @@ class InlineSkuReviews extends BaseComponent
     super(el, assigns)
     {@reviewscore, @reviews_count} = assigns.product_information
 
-  template: -> """
-    <div class="sa-rating-with-count">
-      #{starRating(@reviewscore, @reviews_count, false)}
-    </div>
-    <div class="sa-inline-separator"></div>
-    <div class="sa-inline-secondary-indicators">
-      <div class="sa-reviews-count-texty">
-        <span class="sa-reviews-count-value">#{@reviews_count}</span>
-        <span class="sa-reviews-count-text">@@translations.partner_sku_reviews.product_reviews</span>
+  template: ->
+    if @reviews_count == 1
+      reviews_text = "@@translations.partner_sku_reviews.product_review"
+    else
+      reviews_text = "@@translations.partner_sku_reviews.product_reviews"
```

Comment:
[Style]

Use single quotes and i suggest:

```coffeescript
reviews_text = if @reviews_count == 1
                 '@@translations.partner_sku_reviews.product_review'
               else
                 '@@translations.partner_sku_reviews.product_reviews'
```

Reply:
Yes I started doing this but for some reason I couldn't make it compile :rofl: